server/src/main/java/org/eclipse/openvsx/RegistryAPI.java

```diff
@@ -750,7 +750,7 @@ public ResponseEntity<SearchResultJson> search(
         }
 
         return ResponseEntity.ok()
-                .cacheControl(CacheControl.maxAge(10, TimeUnit.MINUTES).cachePublic())
+                .cacheControl(CacheControl.noCache().cachePublic())
```

Comment:
I wonder if there is a better way of replacing the cache instead of disabling it altogether? cc @amvanbaren 

Otherwise I'm good with the changes